### PR TITLE
Squash PR's in k/cloud-provider-openstack

### PR DESCRIPTION
**Pull request by chaosaffe**
/cc @dims @adisky

### CODE REVIEW

**Comment by adisky** [comment]
@chaosaffe Thanks, it is nice to have it

File: prow/config.yaml [hunk 1]
```diff
@@ -449,6 +449,7 @@ tide:
     kubernetes-sigs/cluster-api-provider-openstack: squash
     kubernetes-sigs/cluster-api-provider-vsphere: squash
     kubernetes-sigs/kubespray: squash
+    kubernetes/cloud-provider-openstack: squash
```
Comment (neolit123) [inline prow/config.yaml]: i made the minor complain that this functionality nukes GPG keys and SOB sign-offs in multi-commit PRs.
repo maintainers should be wary of picky contributors, but if the repo is fine with that SGTM.

Comment (BenTheElder) [reply prow/config.yaml]: :100:
/approve
/hold
please ensure the maintainers are OK with this before merging (`/hold cancel`) :-)

**Pull request merged**